cluster/ci/origin/pipelines/cleanup-image-streams/Jenkinsfile

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env groovy
+pipeline {
+  agent any
+  stages {
+    stage("Cleanup ImageStreams") {
+      steps {
+        script {
+          openshift.withCluster() {
+            def cleanupList = []
+            def currentSecsStr = sh(script: "date +%s", returnStdout: true)
+            def currentSecs = currentSecsStr as Integer
+            def imageStreams = openshift.selector("is")
+            imageStreams.withEach() {
+              def imageStream = it.object()
+              // Only process image streams that are part of a build
+              if (imageStream.metadata.labels != null && imageStream.metadata.labels.template == "output-imagestream") {
+                def creationTimestamp = imageStream.metadata.creationTimestamp
+                def imageStreamCreationSecsStr = sh(script: "date --date=${creationTimestamp} +%s", returnStdout: true)
+                def imageStreamCreationSecs = imageStreamCreationSecsStr as Integer
+                def ageDays = (currentSecs - imageStreamCreationSecs) / 3600 / 24
```

Comment:
Maybe

```sh
$ date --date '2 days ago' +%s
1509818640
```

And a simple comparison between creation time?

Reply:
sgtm, will update